docs/content/execute.md

```diff
@@ -36,14 +36,25 @@ execute:
   execute_notebooks: auto
 ```
 
+or equivalently:
+
+```yaml
+sphinx:
+  config:
+    nb_execution_mode: auto
+```
+
+Both of the above configurations are global, for per-file use `execution_mode` key.
+
 This will only execute notebooks that are missing at least one output.
 If the notebook has *all* of its outputs populated, then it will not be executed.
 
 **To force the execution of all notebooks, regardless of their outputs**, change the
 above configuration value to:
 
 ```yaml
-execute_notebooks: force
+execute:
```

Comment:
@AakashGfude is this configuration managed by `jupyter-book` or `myst-nb` do you know?

Reply:
configurations related to executing notebooks are managed by `myst-nb` .